local/languages/finna/sv.ini

```diff
@@ -78,9 +78,11 @@ All Names = "Alla namnformer"
 Amount of Viewers = "Antal tittare"
 Appraisal = "Informationsvärdering"
 Archive = "Arkiv"
+Archive Authors = "Upphovsmän"
```

Comment:
Näitä käännöksiä ei tarvita, jos laitetaan title context:iin RecordDataFormatterFactoryssa.

Reply:
Toi käännös on Tekijät-yläotsikkoa varten, missään sv-käännöksessä ei ole oikeaa versiota Upphovsmän, vaan Authors=Författarna joka ei ole hyvä käännös

Reply:
Ok. Koska "Archive Authors" ei ole sellaisenaan järkevä näytettävä, voisi avain olla archive_authors.